### PR TITLE
fix: throw error on missing snowpack github token

### PR DESCRIPTION
**Pull request by c-ehrlich**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

- the code block for throwing an error if the SNOWPACK_PUBLIC_GITHUB_TOKEN is missing was inaccessible because the value is either equal to the token itself or 'hello', both of which pass a null check
- might be related to https://github.com/t3-oss/create-t3-app/pull/372

---

## Screenshots

image

💯

### CODE REVIEW

**Review by nexxeln — APPROVED** [review]
Good to have I guess

File: www/src/components/blog/avatarList.astro [hunk 1]
```diff
@@ -18,7 +18,7 @@ type Commit = {
 async function getCommits(url: string) {
   try {
     const token = import.meta.env.SNOWPACK_PUBLIC_GITHUB_TOKEN ?? "hello";
-    if (!token) {
+    if (!token || token === "hello") {
```
Comment (juliusmarminge) [inline www/src/components/blog/avatarList.astro:21]: I think we should probably remove the `?? 'hello'` instead?

Comment (c-ehrlich) [reply www/src/components/blog/avatarList.astro:21]: yeah  i agree

**Comment by juliusmarminge** [comment]
also I don't know if SNOWPACK_PUBLIC is the correct prefix anymore? https://docs.astro.build/en/guides/environment-variables/

**Comment by nexxeln** [comment]
Thanks!

**Pull request merged**